v2/worker/src/jobs.ts

```diff
@@ -45,13 +45,11 @@ export class Job extends BrigadierJob {
         },
       })
       if (response.status != 201) {
-        console.log(response.data)
-        throw new Error(response.data)
+        throw new Error(`Received ${response.status} from the API server`)
```

Comment:
Throwing `response.data` was probably a very dumb thing for me to have done in the first place.

Reply:
Oh yea - good catch.